<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_MerClass</class>
 <widget class="QMainWindow" name="qt_MerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DH_MER</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>351</width>
      <height>721</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>设备控制</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QPushButton" name="startCapture">
          <property name="text">
           <string>开始采集</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="closeDevice">
          <property name="text">
           <string>关闭设备</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="stopCapture">
          <property name="text">
           <string>停止采集</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="openDevice">
          <property name="text">
           <string>打开设备</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>116</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="saveSettings">
          <property name="text">
           <string>保存配置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadSettings">
          <property name="text">
           <string>读取配置</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>模拟控制</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>增益模式</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>白平衡模式</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>白平衡通道</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>白平衡系数</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="balanceRatioSelector">
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>Red</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>Red</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Green</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>自动增益下限</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="gainValueMin">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>自动增益上限</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="balanceRatio">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="balanceWhiteAuto">
          <property name="currentText">
           <string>Continuous</string>
          </property>
          <item>
           <property name="text">
            <string>Continuous</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Off</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Once</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>增益值(0-24dB)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="gainAuto">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Continuous</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Off</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Once</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="gainValue">
          <property name="text">
           <string>16</string>
          </property>
          <property name="maxLength">
           <number>32768</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="gainValueMax">
          <property name="text">
           <string>16</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>基本参数设置</string>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>34</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>曝光时间(us)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>69</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>帧率</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="exposeTime">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>34</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>10000</string>
        </property>
        <property name="placeholderText">
         <string extracomment="请输入&gt;31的数字"/>
        </property>
       </widget>
       <widget class="QLineEdit" name="fps">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>69</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>20</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>设备信息</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>厂商名称</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="factoryName">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>基本型号</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="basicModel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>图像宽度</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="width">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>图像高度</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="height">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>671</width>
      <height>641</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QLabel" name="label_Img">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>640</width>
       <height>512</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>总采集数</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="frameID">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>待保存数</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="frameRemain">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="showButton">
        <property name="text">
         <string>展示图像</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_Mer.qrc"/>
 </resources>
 <connections/>
</ui>
